dev_docs/README-android.md

```diff
@@ -15,4 +15,80 @@ This document will explain a number of different use cases:
 1. How to Run the Sample Application
 2. How to perform simple customizations to the Sample Application
 3. How to perform complex modifications to the Sample Application
-4. How to update an existing Application with the Android Skin
+
+# How to update an existing Application with the Android Skin
+
+1. Copy **<alice>/build/intermediates/bundles/release/classes.jar** to **<myapp>/libs/classes.jar**
+
+2. Rename <myapp>/libs/**classes.jar** to <myapp>/libs/**SkinSDK.jar**
```

Comment:
Customers will more usually download the OoyalaSkinSDK.zip file.  You can see an example of the zip contents here:

https://github.com/ooyala/android-sample-apps/tree/dev/vendor/Ooyala/OoyalaSkinSDK-Android

Can you make step 1 more like, "Download Zip File  <Link to be added>", and step 2 , move the actual jar name to the correct place?